src/proposals/forms/mixins.py

```diff
@@ -130,6 +130,10 @@ class Meta:
                 "(coronavirus) is still raging this August?"
 
             ),
+            'first_time_speaker': _(
+                "Is it the first time to be the speaker of pycon?"
```

Comment:
Note that we only care about whether the submitter has ever given any talk in _PyCon TW_. e.g. If one was a speaker in, say, PyCon JP before, he/she still can be the a _first-time speaker of PyCon TW_. 

```suggestion
                "Is it your first time to be the speaker of PyCon TW?"
```